database/storage.go

```diff
@@ -45,19 +45,8 @@ SET NAMES utf8;
 SET FOREIGN_KEY_CHECKS = 0;
 
 `
-	var hostname string
-	row := s.conn.QueryRow("SELECT @@hostname")
-	err := row.Scan(&hostname)
-	if err != nil {
-		return "", err
-	}
-
-	var db string
-	row = s.conn.QueryRow("SELECT DATABASE()")
-	err = row.Scan(&db)
-	if err != nil {
-		return "", err
-	}
+	hostname, _ := s.hostname()
```

Comment:
Trust me: You want to handle **every** error (even for testing) , otherwise you will regret it 😅

Reply:
I can't learn this lesson enough times. Ask @rafaeljesus . Haha. Thanks!